services/openshift/scripts/openshift_provision

```diff
@@ -43,23 +43,16 @@ done
 # Create Docker Registry
 if [ ! -f ${ORIGIN_DIR}/configured.registry ]; then
   echo "[INFO] Configuring Docker Registry"
-  oadm registry --create --credentials=${OPENSHIFT_DIR}/openshift-registry.kubeconfig || exit 1
+  oadm registry --create --service-account=registry || exit 1
   oadm policy add-scc-to-group anyuid system:authenticated || exit 1
   touch ${ORIGIN_DIR}/configured.registry
 fi
 
 # For router, we have to create service account first and then use it for router creation.
 if [ ! -f ${ORIGIN_DIR}/configured.router ]; then
   echo "[INFO] Configuring HAProxy router"
-  echo '{"kind":"ServiceAccount","apiVersion":"v1","metadata":{"name":"router"}}' \
-    | oc create -f -
-  oc get scc privileged -o json \
-    | sed '/\"users\"/a \"system:serviceaccount:default:router\",'  \
-    | oc replace scc privileged -f -
-  oadm router --create \
-    --expose-metrics=true \
-    --credentials=${OPENSHIFT_DIR}/openshift-router.kubeconfig \
-    --service-account=router
+  oadm policy add-scc-to-user privileged system:serviceaccount:default:router || exit 1
+  oadm router --service-account=router --expose-metrics=true || exit 1
```

Comment:
@praveenkumar can you please add some comments here, so that it would be easy to read the code when compared to https://docs.openshift.org/latest/install_config/install/docker_registry.html#registry-non-production-use